src/engine/test/source/hlp/hlp_test.cpp

```diff
@@ -120,6 +120,26 @@ TEST(hlpTests_logQL, options_parsing)
     ASSERT_EQ("temp", result["_temp"]);
     ASSERT_EQ("temp1", result["_temp1"]);
     ASSERT_EQ("temp2", result["_temp2"]);
+
+}
+
+TEST(hlpTests, annon_type)
+{
+    const char *logQl = "this has an annon ip <_tempIp/ip> in it";
+    const char *event = "this has an annon ip 127.0.0.1 in it";
```

Comment:
This test is already covered by **IPV4_success**. We should remove it.

Reply:
done